models/news-model.js

```diff
@@ -40,33 +40,32 @@ exports.countComments = () => {
     });
 };
 //merge 6 here
-exports.insertNewUser = (username) => {
-  if (!username) {
-    return Promise.reject({ status: 400, msg: "Error: bad request" });
-  } else {
-    const query =
-      "INSERT INTO users (username, name) VALUES ($1, 'placeholderName') RETURNING *;";
-    return db.query(query, [username]).then((user) => {
-      return user.rows[0];
-    });
-  }
-};
 
 exports.insertNewComment = (newComment, article_id) => {
-  if (Object.keys(newComment).length !== 2) {
-    return Promise.reject({ status: 400, msg: "Error: bad request" });
+  if (Object.keys(newComment).length < 2) {
```

Comment:
It may be more robust to create a white list specifically saying what keys must be in the body.